core/Settings/FieldConfig.php

```diff
@@ -63,6 +63,12 @@ class FieldConfig
     const UI_CONTROL_SINGLE_SELECT = 'select';
 
     /**
+     * Shows an expandable select field which is useful when each selectable value belongs to a group.
+     * To use this field assign it to the `$uiControl` property.
+     */
+    const UI_CONTROL_SINGLE_EXPANDABLE_SELECT = 'expandable-select';
```

Comment:
New UI control that can be used eg in plugin settings... it is a select but with search and categories etc.